src/utils/parser.js

```diff
@@ -0,0 +1,14 @@
+const parseCarName = (carNamesString) => {
+  return carNamesString.split(',').map(carName=>carName.trim());
+}
+
+const parseIteration = (iterationNumberString) => {
+  return parseInt(iterationNumberString, 10);
```

Comment:
```suggestion
  return Number(iterationNumberString);
```
이런식으로 Number()를 사용해보시는 건 어떨까요? 
```
parseInt("12.34") // 12
Number("12.34")   // 12.34

parseInt("12abc") // 12
Number("12abc")   // NaN
```
이런상황에서 좀 더 유용해 보이지않나요? 🤔

Reply:
아 1주차에 Number('') === 0이 되는 문제떄문에 parseInt를 써보려다가 오히려 실수를 한 부분들이 좀 있더라구요. 감사합니다!